app/views/moderations/actions_panel.html.erb

```diff
@@ -110,7 +110,12 @@
 
       <div id="adjust-tags-options" class="adjust-tags-options dropdown-options hidden">
         <% @moderatable.tags.each do |tag| %>
-          <% moderatable_tag = @tag_moderator_tags.pluck(:name).include?(tag.name) || policy(@moderatable).can_adjust_any_tag? %>
+          <% last_added_by_admin = last_adjusted_by_admin?(@moderatable, tag, "addition") %>
```

Comment:
I don't feel great about adding this query in the view but I'm not sure that I see a way around it and we'd like to proactively show when a tag cannot be adjusted by a tag_moderator because it was last adjusted by an admin.

Reply:
> I expect this might be too much scope for this PR, but could make a good follow-up maybe — is that we might want to promote the tag_moderator_tags query into a model or decorator, at some point. But, more relevantly maybe, we wouldn't need to have access to @tag_moderator_tags inside this helper, or even a theoretical policy method here, because we'd be able to get what we needed from moderator_for_tags.)

Thanks @jaw6, this sounds promising - I'll add this to the board and look into it ✨